### PR TITLE
13 model tests

### LINKED ISSUE #13: Create models and tests for objects
- Jeepney
- Building
- Obstacle
- Bonus

Take a look at the code for drawImage regarding attributes we may want to pass in for objects that we would be creating potentially from a sprite sheet or other image source.

### PR DESCRIPTION
**Pull request by allindow**
Close #13 Create models and tests for objects: building, bonus, obstacle

### CODE REVIEW

**Comment by ryanflach** [comment]
@allindow Since all of our sprites are going to be somewhat unrelated (meaning we will likely have individual images rather than a sprite sheet), I think we can remove `sx` and `sy` from each class. Otherwise, this looks fantastic.

**Comment by allindow** [comment]
@ryanflach cool I will remove those and push up again. Just wasn't sure if you had any luck finding sprite sheets full of filipino food ;)

**Comment by allindow** [comment]
@ryanflach ready

File: lib/bonus.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var Bonus = function(img, x, y, sx, sy, width, height){
```
Comment (ryanflach) [inline lib/bonus.js]: Still need to remove `sx` and `sy` from this argument list.

File: lib/building.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+var Building = function(img, x, y, sx, sy, width, height){
```
Comment (ryanflach) [inline lib/building.js]: Still need to remove `sx` and `sy` from this argument list.

File: lib/obstacle.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+var Obstacle = function(img, x, y, sx, sy, width, height){
```
Comment (ryanflach) [inline lib/obstacle.js]: Still need to remove `sx` and `sy` from this argument list.

File: test/bonus_test.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+const chai = require('chai');
+const assert = chai.assert;
+const Bonus = require('../lib/bonus')
+
+describe('Bonus', function(){
+  context('with given attributes', function(){
+    var bonus = new Bonus('bonus.png', 50, 300, 50, 300, 150, 250);
```
Comment (ryanflach) [inline test/bonus_test.js]: Can remove the first 50 and 300 from this argument list (no more `sx`, `sy`)

File: test/building_test.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+const chai = require('chai');
+const assert = chai.assert;
+const Building = require('../lib/building')
+
+describe('Building', function(){
+  context('with given attributes', function(){
+    var building = new Building('building.png', 50, 300, 50, 300, 150, 250);
```
Comment (ryanflach) [inline test/building_test.js]: Can remove the first 50 and 300 from this argument list (no more `sx`, `sy`)

File: test/obstacle_test.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
+const chai = require('chai');
+const assert = chai.assert;
+const Obstacle = require('../lib/obstacle')
+
+describe('Obstacle', function(){
+  context('with given attributes', function(){
+    var obstacle = new Obstacle('obstacle.png', 50, 300, 50, 300, 150, 250);
```
Comment (ryanflach) [inline test/obstacle_test.js]: Can remove the first 50 and 300 from this argument list (no more `sx`, `sy`)

**Review by ryanflach — CHANGES_REQUESTED** [review]
Still need a few removals.

**Review by ryanflach — APPROVED** [review]

**Comment by ryanflach** [comment]
@allindow looks great. Thanks for those quick fixes. Merging.

**Pull request merged**